sequencescape-api.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   POST_INSTALL
 
   s.add_dependency('activemodel', '>= 5.0.0')
-  s.add_dependency('activesupport', '>= 5.0.0', '< 7.0.0')
+  s.add_dependency('activesupport', '>= 5.0.0', '< 6.1.0')
```

Comment:
What was the reason for pinning this? I think limber id already on 6.1 so can't be upgraded to this/